mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
```

Comment:
The error message should be consistent throughout the application. In other parts of the code, error messages are wrapped in errorHandler. This should be consistent here as well. It should be `return res.status(404).json({ error: errorHandler("No such order found") });`